WooCommerce/Classes/Authentication/Prologue/LoginPrologueViewController.swift

```diff
@@ -37,6 +38,7 @@ final class LoginPrologueViewController: UIViewController {
     init(analytics: Analytics = ServiceLocator.analytics,
          featureFlagService: FeatureFlagService = ServiceLocator.featureFlagService) {
         isNewToWooCommerceButtonShown = featureFlagService.isFeatureFlagEnabled(.newToWooCommerceLinkInLoginPrologue)
+        self.isOnboardingFeatureEnabled = featureFlagService.isFeatureFlagEnabled(.loginPrologueOnboarding)
```

Comment:
Nit: `self` can be removed.

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/7291/commits/c7dd9ab4d249c7c36cd770a2275570c43fa9f76d